examples/no_ignition_prefix/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 project(no_ignition_prefix VERSION 0.1.0)
 find_package(ignition-cmake2 REQUIRED)
-ign_configure_project(NO_IGNITION_PREFIX)
+ign_configure_project(
+  INCLUDE_DIR no_ign
+  NO_IGNITION_PREFIX)
```

Comment:
Can you also update the example's `README` to point out the usage of `INCLUDE_DIR` and what it does?

Reply:
yes, I will do it now

Reply:
https://github.com/ignitionrobotics/ign-cmake/pull/190/commits/3edfd842aa95fe31794995a472a058c654877ceb